src/try-cb-lcb.c

```diff
@@ -23,6 +23,27 @@
 #include "try-cb-lcb.h"
 #include "util.h"
 
+#if defined(__linux__)
+#include <kore/seccomp.h>
+
+// syscalls required by libcouchbase
+// See https://docs.kore.io/4.1.0/api/seccomp.html
+// List of syscalls retrieved by running with `seccomp_tracing yes` in config.
+KORE_SECCOMP_FILTER("try-cb-lcb",
```

Comment:
@osfameron When I ran through a quick sanity test, just realized we're missing one:
```
try-cb-api               | [parent]: seccomp violation, [wrk 2] pid=288, syscall=96:gettimeofday
```

Reply:
Good catch, thanks! You must have tested a few more cases than I tried :)
(Or possibly the failure was non-fatal with tracing enabled, so I just missed it? Anyway, adding in.)